cdc_rsync_server/main.cc

```diff
@@ -60,10 +70,19 @@ ServerExitCode GetExitCode(const absl::Status& status) {
 }  // namespace cdc_ft
 
 int main(int argc, const char** argv) {
-  if (argc < 2) {
-    printf("Usage: cdc_rsync_server <port> cdc_rsync_server <size> <time> \n");
-    printf("       where <size> and <time> are the file size and modified\n");
-    printf("       timestamp (Unix epoch) of the corresponding component.\n");
+  if (argc == 2 && strcmp(argv[1], "--version") == 0) {
+    printf("%s\n", BUILD_VERSION);
+    return 0;
+  }
```

Comment:
Is this ever used? If not, maybe just add a line below:
 printf("cdc_rsync_server - Remote component of cdc_rsync. Version: %s", BUILD_VERSION);

Reply:
This is not used, it's just to let users to check which versions they are using.
Can print it unconditionally.